README.md

```diff
@@ -38,82 +38,37 @@ npm run test
-
-#### Additions
-
-> For example, adding a new spec or subsection via a new or updated IDL file
-
-- Does the new objects or fields show up in [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data)? If not, it's likely too soon.
-- Is the IDL source from WHATWG?
-    - Are the additions available in at least two of [Firefox](https://searchfox.org/mozilla-central/search?q=&path=), [Safari](https://webkit-search.igalia.com/webkit/search?q=&path=) and Chromium?
-- Is the IDL source from W3C?
-    - What stage of the [W3C process](https://en.wikipedia.org/wiki/World_Wide_Web_Consortium#Specification_maturation) is the proposal for these changes: We aim for Proposed recommendation, but can accept Candidate recommendation for stable looking proposals.
-    - If it's at Working draft the additions available in all three of Firefox, Safari and Chromium
-- Could any types added at the global scope have naming conflicts?
-- Are the new features going to be used by a lot of people?
-
-#### Removals
-
-> For example, removing a browser-specific section of code
-
-- Do the removed objects or fields show up in [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data)? If so, are they marked as deprecated?
-- Does an internet search for the fields show results in blogs/recommendations?
-- When was the deprecation (this can be hard to find) but was it at least 2 years ago if so?
-
-## Build Instructions
+If you are familiar with Web IDL, you may also want to check whether the upstream IDL itself can be made more specific. Doing so will reduce the need for manual overrides in this repository and thus can be more helpful.
 
 # This repo
 
 ## Code Structure
 
-- `src/index.ts`: handles the emitting of the `.d.ts` files.
+- `src/build.ts`: handles the emitting of the `.d.ts` files.
 - `src/test.ts`: verifies the output by comparing the `generated/` and `baseline/` contents.
 
 ## Input Files
 
-- `browser.webidl.preprocessed.json`: a JSON file generated by Microsoft Edge. **Do not edit this file**.
-    - Due to the different update schedules between Edge and TypeScript, this may not be the most up-to-date version of the spec.
 - `mdn/apiDescriptions.json`: a JSON file generated by fetching API descriptions from [MDN](https://developer.mozilla.org/en-US/docs/Web/API). **Do not edit this file**.
-- `addedTypes.json`: types that should exist in either browser or webworker but are missing from the Edge spec. The format of the file mimics that of `browser.webidl.preprocessed.json`.
-- `overridingTypes.json`: types that are defined in the spec file but has a better or more up-to-date definitions in the json files.
-- `removedTypes.json`: types that are defined in the spec file but should be removed.
+- `addedTypes.jsonc`: types that should exist but are missing from the spec data.
+- `overridingTypes.jsonc`: types that are defined in the spec but has TypeScript-friendly modifications in the json files.
```

Comment:
It can't be "more up to date" anymore since the data itself is being constantly autoupdated while the JSON file is handled manually. Removing that word makes it "have better definitions" which is a vague expression.

Reply:
```suggestion
- `overridingTypes.jsonc`: types that are defined in the spec but have TypeScript-friendly modifications in the json files.
```